build.gradle

```diff
@@ -69,13 +71,18 @@ task checkJavaVersion {
 
 // Configure all Java projects
 configure(projectsWithFlags('java')) {
+    apply plugin: 'net.ltgt.errorprone'
+
     // Common dependencies
     dependencies {
         // All projects currently require ':core' (except itself)
         if (project != project(':core')) {
             api project(':core')
         }
 
+        // Error Prone compiler
+        errorprone 'com.google.errorprone:error_prone_core'
```

Comment:
The default value of `disableWarningsInGeneratedCode` is `false`.
How about making it true for fewer noisy.

Reply:
I couldn't feel any differences with and without warning. So I think we can just leave it as it is.

Reply:
OK, SGTM